penguin/__init__.py

```
@@ -175,6 +138,60 @@ def __getattr__(name):
+                "Engine": Engine,
+                "EngineSettings": EngineSettings,
+            }
+        )
+        return _optional_exports[name]
+
+    if name == "PenguinAgent":
+        try:
+            from .agent import PenguinAgent
+
+            _optional_exports["PenguinAgent"] = PenguinAgent
+            return PenguinAgent
```

Ruff (0.15.15)

[warning] Consider moving this statement to an `else` block

(TRY300)